src/renderers/stateRenderer.js

```diff
@@ -157,6 +235,48 @@ function getRegions() {
     });
 }
 
+function getFacilities() {
+    ipcRenderer.send("Facility:getFacilitiesByState", parseInt(getProcessArgObj()));
+    ipcRenderer.once("Facility:getFacilitiesByStateOK", (e, res) => {
+        $('#listOfFacilities').empty();
+        let table = `<table id="state-facility" class="table table-striped table-bordered table-sm" cellspacing="0" width="100%">
```

Comment:
Let's extract this like we talked about

Reply:
You want me to make another function. For table creation?